src/Zynga/Framework/PgData/V1/PgModel.hh

```diff
@@ -35,13 +35,19 @@ use Zynga\Framework\PgData\V1\SqlGenerator;
 use \Exception;
 
 abstract class PgModel implements PgModelInterface {
+
+  protected bool $_allowWriterOverride = true;
```

Comment:
Lets add a comment here explaining when this is used. The name can be confusing as it might mean that we will use writer for everything but this will only be used for resultSetCacheOverride.

maybe allowWriterOnClearingResultSetCache or something more specific ? 

Also default this to false to release the code and then we can hotfix that to true.